ddtrace/tracer.py

```diff
@@ -145,22 +145,31 @@ def __init__(
         self.writer = writer  # type: TraceWriter
 
         # TRACER_PARTIAL_FLUSH_ENABLED should be deprecated after version 1.0.0 is released
-        partial_flush_enabled_default = asbool(get_env("tracer", "partial_flush_enabled", default=False))
+        partial_flush_enabled_deprecated_config = asbool(get_env("tracer", "partial_flush_enabled", default=False))
+        if partial_flush_enabled_deprecated_config:
```

Comment:
_this is an aside, not suggesting you should make this change._

what happens if they set `DD_TRACER_PARTIAL_FLUSH_ENABLED=false`, should we still log? the way it is here we will not log a warning.

Reply:
I decided to only log a warning if DD_TRACER_PARTIAL_FLUSH_ENABLED was set to true. Since the default value is false this case `DD_TRACER_PARTIAL_FLUSH_ENABLED=false` would result in the same behavior after the key is no longer supported. Although it could cause confusion if a customer sets this flag to false now and then sets it to true after version 1.0.0 is released

Reply:
we probably need to do a check like:

```python
default_value = False
if "DD_TRACER_PARTIAL_FLUSH_ENABLED" in os.environ:
    deprecation("DD_TRACER_.... use DD_TRACER instead", version="1.0.0")
    default_value = asbool(os.environ["DD_TRACER_PARTIAL_FLUSH_ENABLED"])

self._partial_flush = asbool(getenv("...", default=default_value))
```

(ish)

wdyt?

Reply:
Done